### PR TITLE
Bump Rack to the latest version for security fixes

### PR DESCRIPTION
**Pull request by issyl0**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org/'
 
-gem 'activerecord', '4.1.5' # Ideally version should be synced with Transition
+gem 'activerecord', '4.1.6' # Ideally version should be synced with Transition
```
Comment (jennyd) [inline Gemfile]: 4.1.11 is the latest version of ActiveRecord, released yesterday: https://rubygems.org/gems/activesupport/versions

Comment (issyl0) [inline Gemfile]: Bleh. I read the "version should be synced with Transition" too literally! Done.

File: Gemfile [hunk 2]
```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org/'
 
-gem 'activerecord', '4.1.5' # Ideally version should be synced with Transition
+gem 'activerecord', '4.1.11' # Ideally version should be synced with Transition
 gem 'pg', '0.17.1'
 gem 'nokogiri', '1.6.0'
-gem 'rack', '1.5.2'
+gem 'rack', '1.6.2'
```
Comment (jennyd) [inline Gemfile]: There are a [lot of changes between rack 1.5.2 and 1.6.2](https://github.com/rack/rack/compare/1.5.2...1.6.2) - going to 1.5.4 today would be safer.

Comment (issyl0) [inline Gemfile]: I did this originally before I realised that 1.5.4 was allowed. Changed.

**Pull request merged**